dev-system/support/build-for-container.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# Build and run the CMR as a single dev-system or as a set of microservices
+# Parameters:
+#  - separate: build and run each microservice separately
+#  - together: build and run dev-system in a container
+#  - default: pass no parameters, and the script will build and run dev-system
+
+build-and-run-container () {
```

Comment:
This should take one more argument - port number.

Then when you run you would say something like `docker run -d $1 -p $2:$2

Reply:
I thought `EXPOSE` statements in the Dockerfile did that for you, but I've updated that anyway for good measure

Reply:
From what I saw EXPOSE will force those ports to be mapped to a random port number on your localhost. (somewhere in the 3xxxx range). I had to explicitly set the ports I wanted mapped to my localhost port in order to make it work.

Reply:
Not exactly - that just makes it so that you _can_ expose the port to the outside world.  It works in conjunction with the, say `-P` flag in `docker run` or `-p <exposed port>` or `-p <exposed port>:<host opened port mapped>`